content/rookie-fitting/amarr.md

```diff
@@ -9,7 +9,28 @@
 
 The Amarr corvette is the Impairor.
 
-<object type="image/svg+xml" data="https://o.smium.org/api/convert/119443/svg/119443-alpha-impairor-wdrone.svg?privatetoken=9063189616642752512"><a href="https://o.smium.org/loadout/private/119443/9063189616642752512">View Alpha Impairor w/Drone on Osmium</a></object>
+### EFT
+[Impairor, Impairor alpha]
+
+Gatling Pulse Laser I
```

Comment:
Let's tighten up the gaps here - I think ideally for compact, easy-to-read fits, we want only 4 blank lines:  1 after the fit title, one between high and mid slots, one between mid and low slots, and one between low slots and ammo/drones. So this fit would wind up spaced like this:

```
[Impairor, Impairor alpha]

Gatling Pulse Laser I
Gatling Pulse Laser I

1MN Afterburner I
Stasis Webifier I

Small Armor Repairer I
Damage Control I

Acolyte I x1
Multifrequency S x2
```

Make sense?

Reply:
ahhh! i was wondering how to neaten all that up! makes perfect sense! :)